pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1161,3 +1166,29 @@ type RestoreStatus struct {
 	TimeCompleted metav1.Time        `json:"timeCompleted"`
 	Conditions    []RestoreCondition `json:"conditions"`
 }
+
+// +k8s:openapi-gen=true
+// DashBoard describe the TiDB Dashboard desired state
+type DashBoardSpec struct {
+	// PDDashboardIngress describe the ingress of the TiDB Dashboard
+	// +optional
+	Ingress *IngressSpec `json:"ingress,omitempty"`
```

Comment:
it is possible to use Kubernetes builtin ingress types directly?

Reply:
I suggest not as we should control the path that the ingress would only expose the dashboard api excluding pdapi.